src/Lock.php

```diff
@@ -535,9 +705,9 @@ public static function showForItem(CommonDBTM $item)
      **/
     public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
     {
-
         if ($item->isDynamic() && $item->can($item->fields['id'], UPDATE)) {
-            return Lock::getTypeName(Session::getPluralNumber());
+            $nb = Lock::countLockFromItem($item);
```

Comment:
All these queries should not be done if user does not want to show counters on tabs.
```suggestion
            $nb = ($_SESSION['glpishow_count_on_tabs']) ? Lock::countLockFromItem($item) : 0;
```